src/app/account.service.ts

```diff
@@ -71,26 +73,31 @@ export class AccountService {
     }
   }
 
-  // Modifiers
-
-  addUser(userInfo: PrivateUserInfo): string {
-    const privateUsers = this.getPrivateUsersRaw();
-    const privateKey = this.cryptoService.seedHexToPrivateKey(userInfo.seedHex);
-    const publicKey = this.cryptoService.privateKeyToBitcloutPublicKey(privateKey, userInfo.network);
-
-    privateUsers[publicKey] = userInfo;
-
-    localStorage.setItem(AccountService.usersStorageKey, JSON.stringify(privateUsers));
-
-    return publicKey;
+  // Public Modifiers
+
+  addUser(keychain: HDKey, mnemonic: string, extraText: string, network: Network, google?: boolean): string {
+    const seedHex = this.cryptoService.keychainToSeedHex(keychain);
+    const btcDepositAddress = this.cryptoService.keychainToBtcAddress(keychain, network);
+    const ethDepositAddress = this.cryptoService.keychainToEthAddress(keychain);
```

Comment:
It's fine, but curious: Is there a reason we moved the pubkey generation for BTC/ETH to identity from backend where it was before?